tokens/debio.json

```
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "",
+    "near_address": "dbio.near",
+    "aurora_address": "0x72f9fedef0fb0fc8db22453b692f7d5a17b98a66",
+    "name": "DeBio Network Token",
```

This should match NEP-141 metadata.

```suggestion
    "name": "Debio,
```